### ISSUE TITLE
s905w - reboots instead of shutting down

### ISSUE THREAD

**Issue by giovEra** [ophub/amlogic-s9xxx-armbian#1461]
fixed

**Comment by giovEra** [comment]
Please tell me if and what information I have to provide.
Thank you

**Comment by ophub** [comment]
`sudo poweroff`

**Comment by giovEra** [comment]
I extracted the dtb from the OTA file, maybe it is useful.
I have uploded the dtb and the dts (converted by me) here: [GTMedia DTB](https://mega.nz/folder/KWYihYIQ#iYVnt5_9D985KdoBq-ePAQ)

**Comment by giovEra** [comment]
Based on the label on the back of the circuit board, it appears to be a clone of: "Tanix TX3 PRO Amlogic S905W"

[![back.jpg](https://i.postimg.cc/YC79WQ8c/back.jpg)](https://postimg.cc/1fWsxnLM)

**Comment by giovEra** [comment]
I've done some research and it appears that the "GTMedia G1" is a clone of the "TX3 PRO S905W" (not made by tanix) which itself is a clone of the "x96 mini".
In fact the "GTMedia G1" and the "x96 mini" have the same ports and in the same positions and neither of them have the vfd.

So I tried all dtb for s905x available in file "Armbian_23.08.0_amlogic_s905w_jammy_5.15.115_server_2023.06.05.img.gz", but always same thing happens: command ``sudo shutdown -h now`` and command ``sudo poweroff`` they cause the tv box to reboot and not shut down.

I also tried to use a new sd card but the problem still occurs.

@ophub can it be a dtb problem? Can you take a look at it?
([android dtb](https://mega.nz/folder/KWYihYIQ#iYVnt5_9D985KdoBq-ePAQ))

**Comment by giovEra** [comment]
I have also tried 5.4.x, 5.10.x, 5.15.x and 6.1x kernels, but no luck.

**Comment by giovEra** [comment]
Hi @ophub,

I have tried "[Armbian_20.10_Arm-64_focal_current_5.9.0.img.xz](https://mega.nz/file/nHgmiZyR#c4b4NxDRx4YVQWoP5zqh5-OD8X1sKgfnRvTwqjtW6sI)", an old armbian build by @150balbes, and the ``sudo poweroff`` command works correctly causing the tv box to turn off. The only problem is that the led remains blue and does not turn red after shutdown.

I tried to compare the dts (convert from dtb "meson-gxl-s905w-p281.dtb") taken from your image and 150balbes' one, and there doesn't seem to be any significant difference.

I leave you the link to the dtb extracted from the image of 150balbes: [dtb](https://mega.nz/folder/LLhCEYKY#vMyL5ukXBhpJunfr6IAGxA)

Maybe the problem is in the kernel?
